test/issues/issues-test.js

```diff
@@ -200,4 +200,61 @@ describe("issues", function () {
             assert(firstFake !== secondFake);
         });
     });
+
+    describe("#1274 - spy.withArgs(args...).callCount is incorrect", function () {
```

Comment:
Although you're fixing a bug, I believe these tests should go in their respective test files (spy, stub, etc.)